scripts/exregional_run_fcst.sh

```diff
@@ -168,6 +168,7 @@ esac
 #
 #-----------------------------------------------------------------------
 #
+HH=$( echo $cdate | cut -c9-10 )
```

Comment:
HH is already defined as an environment variable from the rocoto run_fcst task:
    `<envar><name>HH</name><value><cyclestr>@H</cyclestr></value></envar>`
I think removing this line will not affect the workflow.

FYI, if you want to extra the hour info, you may also use `HH=${cdate:8:2} `

Reply:
Thank you @guoqing-noaa That line is removed.